controller/RestController.php

```diff
@@ -1063,4 +1063,53 @@ public function related($request)
         $ret = $this->transformPropertyResults($request->getUri(), $request->getLang(), $related, "related", "skos:related");
         return $this->returnJson($ret);
     }
+
+    /**
+     * Used for querying new concepts in the vocabulary
+     * @param Request $request
+     * @return object json-ld wrapped list of changed concepts
+     */
+    public function newConcepts($request)
+    {
+        return $this->changed($request);
+    }
+
+    /**
+     * Used for querying modified concepts in the vocabulary
+     * @param Request $request
+     * @return object json-ld wrapped list of changed concepts
+     */
+    public function modifiedConcepts($request)
+    {
+        return $this->changed($request, 'dc:modified');
+    }
+
+    /**
+     * Used for querying changed concepts in the vocabulary
+     * @param Request $request
+     * @return object json-ld wrapped list of changed concepts
+     */
+    private function changed($request, $prop='dc:created')
+    {
+        $vocab = $request->getVocab();
+        $offset = ($request->getQueryParam('offset') && is_numeric($request->getQueryParam('offset')) && $request->getQueryParam('offset') >= 0) ? $request->getQueryParam('offset') : 0;
+        $changeList = $vocab->getChangeList($prop, $request->getLang(), $offset);
+
+        $simpleChangeList = array();
+        foreach($changeList as $conceptInfo) {
+            if (array_key_exists('date', $conceptInfo)) {
+                $simpleChangeList[] =  array( 'uri' => $conceptInfo['uri'],
+                                               'prefLabel' => $conceptInfo['prefLabel'],
+                                               'date' => $conceptInfo['date']->format("Y-m-d\TH:i:sO") );
+            }
+        }
+        return $this->returnJson(array_merge_recursive($this->context,
+                                                        array('@context' => array( '@language' => $request->getLang(),
+                                                                                     'prefLabel' => 'skos:prefLabel',
+                                                                                     'xsd' => 'http://www.w3.org/2001/XMLSchema#',
+                                                                                     'date' => array( '@id' => 'http://purl.org/dc/terms/date', '@type' => 'http://www.w3.org/2001/XMLSchema#date') )
```

Comment:
The datatype is set to `xsd:date`, but it seems to me that the actual values are formatted as `xsd:dateTime` (see format string above). So maybe the `@type` should be changed to `xsd:dateTime` ?

Reply:
Fixed. Originally my intention was to support both dates and dateTimes (one being subclass of the other), but as now the timestamps are formatted all as dateTimes, I changed this accordingly to dateTime.